src/test/java/core/basesyntax/service/impl/ReportWriterToFileServiceToCsvImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.service.impl;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exeptions.WrongExtensionFile;
+import core.basesyntax.service.ReportWriterToFileService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("WriterToCsvImpl Test")
+class ReportWriterToFileServiceToCsvImplTest {
+    private static ReportWriterToFileService WRITER_TO_FILE;
```

Comment:
I would recommend rename this variable simple as `service` , because we test only one service in test class.  You can use this principle in other test classes also. 
Also test naming best practice examples are `testInstance` or `sut` - system under test.